timescaledb/getting-started/create-cagg/create-cagg-basics.md

```diff
@@ -0,0 +1,156 @@
+
+The `CREATE MATERIALIZED VIEW` command triggers the database to create a materialized view with 
+the given name, in this case `stock_candlestick_daily`. In the next line, 
+`WITH (timescaledb.continuous)` instructs TimescaleDB to create a continuous 
+aggregate and not just a generic materialized view. Finally, the query from above
+is added after the `AS` keyword. 
+
+<procedure>
+
+1. Using the aggregate query from above, create a continuous aggregate for daily
+   candlestick data.
+	```sql
+	CREATE MATERIALIZED VIEW stock_candlestick_daily
+	WITH (timescaledb.continuous) AS
+	SELECT
+		time_bucket('1 day', "time") AS day,
+		symbol,
+		max(price) AS high,
+		first(price, time) AS open,
+		last(price, time) AS close,
+		min(price) AS low
+	FROM stocks_real_time srt
+	GROUP BY day, symbol;
+	```
+
+</procedure>
+
+Notice that the `SELECT` statement is the same query you wrote earlier without the 
+`ORDER BY` clause. By default, this code both creates the aggregate 
+*and* materializes the aggregated data. That means the view is created *and* populated 
+with the aggregate calculations from your existing hypertable data.
+
+  <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/continuous-aggregate.jpg" alt="Continuous aggregate upon creation"/>
+
+The query may take some time to run because it needs to perform these calculations
+across all of your stock trade data the first time. After the calculation results 
+are stored, querying the data from the continuous aggregate is much faster. 
+
+Run the following query to get all the data in your continuous aggregate, and note 
+how much faster this is than running the aggregate `SELECT` query on the raw hypertable data.
+
+```sql
+SELECT * FROM stock_candlestick_daily
+ORDER BY day DESC, symbol;
+```
+
+## Real-time continuous aggregates
+By default, all continuous aggregates are created as *real-time* aggregates. This
+means that TimescaleDB always appends (or `UNION`) recent data that has not yet been materialized
+through a refresh policy to the output of the continuous aggregate. In the diagram 
+above that corresponds to the last three points of raw data, which belong to an 
+incomplete bucket. 
+
+<highlight type="note">
+To inspect details about a continuous aggregate, such as its 
+configuration or the query used to define it, use the following 
+informational view:
+
+```sql
+SELECT * FROM timescaledb_information.continuous_aggregates;
+```
+**Results:**
+```bash
+hypertable_schema|hypertable_name |view_schema|view_name     |view_owner|materialized_only|compression_enabled|materialization_hypertable_schema|materialization_hypertable_name|view_definition                                                                                                                                                                                                                                                |
```

Comment:
Might look better displayed with `\x` on. Written in one long line, the query is quite hard to read.